src/index.js

```diff
@@ -30,30 +30,19 @@ const Container = ({ options }) =>
 const onfidoRender = (options, el, merge) =>
   render( <Container options={options}/>, el, merge)
 
-const stripOneCapture = ({blob, documentType, onfidoId, side}) => {
-  const capture = {id: onfidoId, blob}
-  if (documentType) capture.documentType = documentType
-  if (side) capture.side = side
-  return capture
-}
-
-const stripCapturesHash = captures => mapValues(captures,
-  capture => capture ? stripOneCapture(capture) : null)
+const mapCaptures = captures => mapValues(captures, capture => capture)
 
-const getCaptures = () => mapKeys(stripCapturesHash(events.getCaptures()), key => key + 'Capture')
+const getCapturesKeys = () => mapKeys(mapCaptures(events.getCaptures()), key => key + 'Capture')
 
 function bindEvents (options) {
   const eventListenersMap = {
-    documentCapture: () => options.onDocumentCapture(getCaptures().documentCapture),
-    documentBackCapture: () => options.onDocumentCapture(getCaptures().documentBackCapture),
-    faceCapture: () => options.onFaceCapture(getCaptures().faceCapture),
```

Comment:
What about removing the code that triggers these events?

Reply:
Removed